index.ts

```diff
@@ -0,0 +1,43 @@
+/* Part 2 */
+
+function changeMessage(currNr:number, div: number, previousString :string): string{
```

Comment:
We should be consistent with whitespace. I.e. always have a space after : and never before :. There is a mix here. Also, always have a space before the {

Reply:
I quite like the purpose of this function, nice :)

Reply:
I will pay more attention in the future, thank you!